.github/ISSUE_TEMPLATE/release-checklist.md

```diff
@@ -16,18 +16,18 @@ assignees: ''
 
 - [ ] Go to the Zenodo entry for this project (find the link to the latest Zenodo release on the `README.md` file)
 - [ ] Create a "New version" of it.
-- [ ] Delete all existing files
-- [ ] Copy the reserved DOI to this issue
+- [ ] Get a new DOI for the new release
+- [ ] Copy and paste the reserved DOI to this issue
 - [ ] Update release date
-- [ ] Update version number (make sure there is a leading `v`, like `v1.5.7`)
-- [ ] Update version number in Title (use a leading `v` as well)
+- [ ] Update version number in Title (make sure there is a leading `v`, like `v1.5.7`)
+- [ ] Update version number (use a leading `v` as well)
 - [ ] Add as authors any new contributors who have added themselves to `AUTHORS.md` in the same order
 - [ ] Ensure that the first author is "Fatiando a Terra Project" and others are listed alphabetically by last name
 - [ ] Save the release draft
 
 ## Update the changelog
 
-- [ ] Generate a list of commits between the last release tag and now: `git log HEAD...v1.2.3 > changes.rst`
+- [ ] Generate a list of commits between the last release tag and now: `git log HEAD...v1.2.3 > changes.md`
 - [ ] Edit the list to remove any trivial changes (updates by the bot, CI updates, etc).
 - [ ] Organize the list into categories (breaking changes, deprecations, bug fixes, new features, maintenance, documentation).
```

Comment:
```suggestion
- [ ] Organize the list into categories (breaking changes, deprecations, bug fixes, new features, maintenance, documentation).
- [ ] (Optionally) Sort commits in chronological order (i.e. reverse their order). In Vim this can be done by selecting the lines in Visual mode and running `:!tac`.
```